src/app/OperationalSessionSetup.h

```diff
@@ -233,9 +297,13 @@ class DLL_EXPORT OperationalDeviceProxy : public DeviceProxy,
     Callback::CallbackDeque mConnectionSuccess;
     Callback::CallbackDeque mConnectionFailure;
 
+    OperationalSessionReleaseDelegate * mReleaseDelegate;
+
     /// This is used when a node address is required.
     chip::AddressResolve::NodeLookupHandle mAddressLookupHandle;
 
+    ReliableMessageProtocolConfig mRemoteMRPConfig = GetDefaultMRPConfig();
```

Comment:
I kept it around since the call to `EstablishConnection` in `Connect`, the MRP config is not on the stack. I am not familiar in what cases we would be in that state since this is more ephemeral now so it is possible we can remove it.

Would you be okay with me creating a new issue and following up over there to look into removing `mRemoteMRPConfig` member from `OperationalSessionSetup`, and associated cleanup in calls to `EstablishConnection`

Reply:
> Would you be okay with me creating a new issue 

Not only OK, but would prefer that.  That's why I said "Followup" in my review comment above!

Reply:
Issue https://github.com/project-chip/connectedhomeip/issues/21610 has been created to track